homeassistant/components/enocean/sensor.py

```diff
@@ -214,3 +229,39 @@ def value_changed(self, packet):
         humidity = packet.data[2] * 100 / 250
         self._state = round(humidity, 1)
         self.schedule_update_ha_state()
+
+
+class EnOceanWindowHandle(EnOceanSensor):
+    """Representation of an EnOcean window handle device.
+
+    EEPs (EnOcean Equipment Profiles):
+    - F6-10-00 (Mechanical handle / Hoppe AG)
+    """
+
+    def __init__(self, dev_id, dev_name):
+        """Initialize the EnOcean humidity sensor device."""
+        super().__init__(dev_id, dev_name, DEVICE_CLASS_WINDOWHANDLE)
+        self._rawdata = "no packet received"
+
+    def value_changed(self, packet):
+        """Update the internal state of the sensor."""
+
+        self._rawdata = f"{packet}"
+        action = (packet.data[1] & 0x70) >> 4
+        
+        if action == 0x07:
+            self._state = STATE_CLOSED
+        if (action == 0x06) or (action == 0x04):
+            self._state = STATE_OPEN
+        if action == 0x05:
+            self._state = "tilt"
+            
+        self.schedule_update_ha_state()
+
+    @property
+    def state_attributes(self):
+        attr={}
+        
+        attr["rawdata"] = self._rawdata
```

Comment:
Why do we need to expose the raw data?

Reply:
I added it for debugging purposes and to make sure that e. g. missing sensor state updates are due to receptions errors and not my python coding errors

Reply:
Ok, we use debug logging for that purpose. Please add that instead in the `value_changed` method.

Reply:
I removed the property method "attribute". As there is a debug output in the EnoceanDongle class, another debug output would have been redundant. There is no additional change.